spec/attribute.dd

```diff
@@ -86,6 +87,10 @@ $(GNAME LinkageType):
     $(D Windows)
     $(D System)
     $(D Objective-C)
+
+$(GNAME NamespaceList):
```

Comment:
Is this rule still required now that `extern(C++, "foo", )` allows trailing commas?

Reply:
Hum my grammar is a bit rusty, how would you express it ?
Alternatively feel free to fix it and force push to the remote and ping me for review. AFAICS the CI is happy now.

Reply:
> AFAICS the CI is happy now

Finally! Thanks @Geod24

Reply:
Nevermind, it uses `ConditionalExpression` which doesn't have correspongding list rule.